test/test_pexcept.c

```diff
@@ -0,0 +1,74 @@
+/***************************************************************************
+ * Copyright (C) 2016 - 2022 Patricio Bonsembiante. All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ ***************************************************************************/
+
+#include "unity.h"
+#include "putils/pexcept.h"
+
+PEXCEPT_T uncaught_exception;
```

Comment:
This global variables appear unused, please remove them if that's the case

Reply:
solved :)